packages/fluentui/docs/src/examples/components/Chat/Content/ChatExampleReadStatus.shorthand.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { Chat, ReactionGroupProps, ChatProps } from '@fluentui/react-northstar';
+import { EmojiIcon, LikeIcon, EyeFriendlierIcon } from '@fluentui/react-icons-northstar';
+
+const reactions: ReactionGroupProps['items'] = [
+  { key: 'up', icon: <LikeIcon />, content: '1K' },
+  { key: 'smile', icon: <EmojiIcon />, content: 5 },
+];
+
+const items: ChatProps['items'] = [
+  {
+    attached: 'top',
+    contentPosition: 'end',
+    message: (
+      <Chat.Message reactionGroup={reactions} content="Hello" author="John Doe" timestamp="Yesterday, 10:15 PM" mine />
+    ),
+    key: 'message-1',
+  },
+  {
+    attached: 'bottom',
+    contentPosition: 'end',
+    key: 'message-2',
+    message: (
+      <Chat.Message
+        reactionGroup={[{ key: 'up', icon: <LikeIcon />, content: '8' }]}
+        content="I'm back!"
+        author="John Doe"
+        timestamp="Yesterday, 10:15 PM"
+        readStatus="Read by All"
+        readStatusIndicator={<EyeFriendlierIcon size="small" />}
```

Comment:
Should we have this by default as a `backgroundImage` as we do with the other components? If content or children is specified than we won't be adding it

Reply:
Not really sure to be honest since not the default message wouldn't have it I think, just the last messages in the chat.

Reply:
I don't understand why that is a problem. This would just be default style. If users specify object or title inside the shorthand we would render it otherwise no. Isn't that how it should work?

Reply:
@mnajdova to be honest I don't think that `backgroundImage` is a good approach to go as it causes issues like this: https://github.com/microsoft/fluentui/pull/13486

Reply:
I'll also support the `icon` and not `backgroundImage` approach :) 
For me this item is more like an additional slot, so I'd rather not have anything there by default, and leave it to the consumer

@mnajdova , please WDYT?

Reply:
I agree, now that the `<Icon />` component is not that heavy, I guess default icons is a better approach 👍